agent/agent.go

```diff
@@ -173,6 +173,9 @@ func (a *Agent) Run() error {
 	if err := v2.ValidateName(a.config.AgentName); err != nil {
 		return fmt.Errorf("invalid agent name: %v", err)
 	}
+	if timeout := a.config.KeepaliveTimeout; timeout > 0 && timeout < 5 {
```

Comment:
Can a KeepaliveTimeout be 0 seconds? That would essentially disable failing keepalives right?

Reply:
The minimum value is 5 seconds, and we don't support disabling keepalives currently.

Reply:
After chatting with Nikki I realized that the condition should only check for `timeout < 5`